main.tf

```
@@ -57,6 +57,36 @@ resource "aws_iam_service_linked_role" "default" {
   description      = "AWSServiceRoleForAmazonElasticsearchService Service-Linked Role"
 }
 
+# Role that pods can assume for access to elasticsearch and kibana
+resource "aws_iam_role" "elasticsearch_user" {
+  count              = "${var.enabled == "true" ? 1 : 0}"
+  name               = "${module.label.id}-user"
+  assume_role_policy = "${data.aws_iam_policy_document.assume_role.json}"
+  description        = "User of Elasticserach ${module.label.id} cluster"
```

```suggestion
  description        = "IAM Role to assume to access the Elasticsearch cluster"
```

our README generator will not evaluate the interpolation